src/client/src/components/Quiz/BasicQuestion/Answers/Answer/Answer.css

```diff
@@ -0,0 +1,12 @@
+.answer-container{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    flex-grow: 1;
+}
+
+.answer-btn {
+    width: 90%;
+    height: 80% !important;
```

Comment:
Can we avoid "important" somehow?

Reply:
Yes, I replaced it with an inline styling